include/aos/common/tools/time.hpp

```diff
@@ -12,37 +12,53 @@
 #include "aos/common/tools/log.hpp"
 #include "aos/common/tools/string.hpp"
 
+#include <time.h>
+
 namespace aos {
 namespace time {
 
 /**
- * Base type for a time duration. Can also be negative to set a point back in time.
+ * Base type for a time duration in nanoseconds. Can also be negative to set a point back in time.
  */
 using Duration = int64_t;
 
+/**
+ * Nanoseconds to seconds ratio.
+ */
+constexpr int64_t cNanoRatio = 1000 * 1000 * 1000;
+
 /**
  * Returns time duration in years.
  *
  * @param num number of years.
  * @result Duration.
  */
-constexpr Duration Years(int num)
+constexpr Duration Years(int64_t num)
 {
-    (void)num;
-    return 0;
+    constexpr Duration year = 31556925974700000;
+
+    return year * num;
 }
 
 /**
  * An object specifying a time instant.
  */
 class Time {
 public:
+    /**
```

Comment:
I doubt about adding additional namespace for Time. All other Aos tools doesn't have it. And it looks ok for me to have just: aos::Time, aos::Duration instead of aos::time::Time, aos::time::Duration.

Reply:
yes, it makes sense. Done in a separate commit.